docs/install-overview.md

```diff
@@ -0,0 +1,130 @@
+-   Storage element (pick one):
+    -   GridFTP
+        -   [Install standalone OSG GridFTP](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallOSGGridFTP): GridFTP server
+        -   (optional) [Install load-balanced OSG GridFTP](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/LoadBalancedGridFTP): when a single GridFTP server isn't enough
+    -   BeStMan
+        -   [BeStMan Overview](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/BestmanOverview): Bestman-related information, planning, and guides
+        -   [Install Bestman SE](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallOSGBestmanSE): BeStMan2 SRM server + GridFTP server
+        -   [Install Bestman Gateway Hadoop](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallHadoop200SE): BeStMan2 SRM server + GridFTP server + Hadoop
+    -   Hadoop Distributed File System (HDFS)
+        -   [Hadoop Overview](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/HadoopOverview): HDFS information, planning, and guides
+    -   XRootD
+        -   [XRootd Overview](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/XrootdOverview): XRootD information, planning, and guides
+        -   [Install Xrootd Server](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallXrootd): XRootD redirector installation
+        -   [Install BeStMan-Gateway XRootD](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallBestmanXrootdSE): BeStMan2 SRM server + GridFTP server + XRootD fuse
+
+Test OSG Software
+-----------------
+
+At very least, it is vital to test *manual* submission of jobs from inside and outside of your site through your CE to your batch system. If this process does not work manually, it will probably not work for the glideinWMS pilot factory either.
+
+-   [Test job submission into an HTCondor-CE](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SubmittingHTCondorCE)
+-   [OSG Troubleshooting guide](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/TroubleshootingGuide)
+-   [Validating Supported VOs](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/ValidateSupportedVos)
+
+Start GlideinWMS Pilot Submissions
+----------------------------------
+
+To begin running [GlideinWMS](http://www.uscms.org/SoftwareComputing/Grid/WMS/glideinWMS/) pilot jobs at your site, e-mail <osg-gfactory-support@physics.ucsd.edu> and tell them that you want to start accepting Glideins. Please provide them with the following information:
+
+-   The type of CE (HTCondor-CE or the now-unsupported GRAM-CE)
+-   The fully qualified hostname of the CE
+-   Resource/WLCG name
+-   OS major version of your worker nodes — EL 6, EL 7, or a mix of both?
+-   Do your worker nodes use gLExec?
```

Comment:
For new sites / entries, we shouldn't mention `glexec`.  If it's not too much bother, just ditch this line.  We should keep the `glexec` documentation itself for OSG 3.3 users.